addons/udes_stock_picking_batch/views/stock_picking_batch.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0"?>
+<odoo>
+    <data>
+
+        <!-- Stock picking batch form -->
+        <record id="stock_picking_batch_form" model="ir.ui.view">
+            <field name="name">udes_stock.stock.picking.batch.form</field>
+            <field name="model">stock.picking.batch</field>
+            <field name="inherit_id"
+                   ref="stock_picking_batch.stock_picking_batch_form"/>
+            <field name="arch" type="xml">
+                <xpath expr="//field[@name='state']" position="replace">
+                    <field name="state" widget="statusbar" statusbar_visible="waiting,ready,in_progress,done"/>
+                </xpath>
+                <xpath expr="//button[@name='action_confirm']" position="after">
+                    <button name="mark_as_todo" string="Mark as Todo"
+                            confirm="Are you sure you want to mark this batch as todo?" type="object" states="draft"/>
+                </xpath>
+                <xpath expr="//button[@name='action_cancel']" position="replace">
+                    <button name="action_cancel" string="Cancel" type="object" states="ready,waiting,in_progress"/>
+                </xpath>
+            </field>
+        </record>
+
+        <!-- Stock picking batch filter -->
+        <record id="stock_picking_batch_filter" model="ir.ui.view">
+            <field name="name">udes_stock.stock.picking.batch.filter</field>
```

Comment:
Nit: udes_stock i think was a copy paste from udes_stock module. Not to get confused i think would be better to change the name.
```suggestion
            <field name="name">stock.picking.batch.filter.inherit</field>
```
I would use like this as the name of the module is very long

Reply:
Personally I think it is good practice to include the module name (although better at the end imo, so it is appended to the original name)

Helps with debugging when you have an issue with views.